project/src/store/actions.js

```diff
@@ -4,12 +4,19 @@ export const ActionType = {
   CHANGE_CURRENT_CITY: 'changeCurrentCity',
   CHANGE_SORT_TYPE: 'changeSortType',
   TOGGLE_AUTH: 'toggleAuth',
+  DOWNLOAD_OFFERS: 'downloadOffers',
```

Comment:
Группируй по 3 экшены на 1 действие. Request, Success, Error. Название должно быть одно и то же иначе сейчас у тебя дублируются два экшена. Префиксы не забывай
```suggestion
  LOAD_OFFERS_SUCCESS: 'data/loadOffersSuccess',
```